.circleci/config.yml

```diff
@@ -43,7 +43,7 @@ jobs:
           command: "cargo build"
       - run:
           name: "Tests"
-          command: "cargo test && bash .circleci/run_tests.sh && .circleci/generate_coverage.sh"
+          command: "cargo build && bash .circleci/run_tests.sh &&  cargo test && .circleci/generate_coverage.sh"
```

Comment:
The rust integration test expects databases to be set up so we run it after the bash script runs.